experimental/examples/chat-mpp/shared/src/commonMain/kotlin/Messages.kt

```diff
@@ -59,8 +65,9 @@ private inline fun ChatMessage(isMyMessage: Boolean, message: Message) {
             elevation = 8.dp
         ) {
             Box(
-                Modifier.background(brush = Brush.horizontalGradient(listOf(Color(0xff8888ff), Color(0xffddddff))))
-                    .padding(10.dp),
+                Modifier.background(brush = Brush.horizontalGradient(
+                    ChatColors.GRADIENT_2.map { Color(it) })
```

Comment:
I'm guessing this is because you're reusing the same colors from the iOS side, so you can't keep them wrapped in `Color` objects by default?

Reply:
Yes, it's easier to share with iOS.
Actually, we can use multiplatform Color in iOS, but we still need to extract hex value from it.